### ISSUE TITLE
Rich input is not working properly on android

### ISSUE THREAD

**Issue by Insality** [Insality/druid#288]
@Insality I just updated Druid to latest and now the rich input is not working properly on android.
When I type, its not appending new letters.

I also build druid example and it has the same problem

https://discord.com/channels/250018174974689280/1273208001024753714/1310570711353393193

**Comment by blendjam** [comment]
Same here. Works on IOS and Desktop but not on Android.

**Comment by FREEZX** [comment]
Upon further analysis, it seems like this is an known issue defold has with mobile browsers, so in my case I changed the input component to use this package for mobile:
https://github.com/mchlkpng/defold-mobilehtml5-typing
Hope this helps someone

**Comment by blendjam** [comment]
Why is it not working for me? and are the radio_group and check_box component removed or will they be back on full release?
https://github.com/user-attachments/assets/7cc12f47-c203-4457-99da-c5641a6d3a4c

**Comment by Insality** [comment]
@blendjam hello! Please check the input bindings in your `game.project`, try to use `all.input_bindings" and check the text input is working

If true, you need to add a `text` input binding

Checkbox and radio group was removed, since their usage seems way harder than using a button for user (https://github.com/Insality/druid/blob/master/example/examples/basic/checkbox/checkbox.lua)

or just a callback with a enable/disable inner checkmark

**Comment by blendjam** [comment]
@Insality  Hello, Thanks I didn't have all.input_binding in my old project but it used to work. Now adding all.input_binding works. RichInput works on HTML5 and Android but in iOS when holding down the back buck it gets stuck.

https://github.com/user-attachments/assets/7fbec77f-b8aa-4909-a261-8c58074489a4

**Comment by blendjam** [comment]
Hello!  My old project didn't have `all.input_binding` in it's game.project but still it was working xD. Adding `all.input_binding` fixed the problem. The RichInput works on HTML and Android but in iOS the cursor is stuck when holding down back button.

https://github.com/user-attachments/assets/fc0d7da8-aa64-49cb-a43a-35ecff467bf4

**Comment by Insality** [comment]
Not sure what you mean. You mean it not repeat the deletion while holding delete? If true, probably should be handled somehow differently, but in general this is okay for in-game input

**Comment by blendjam** [comment]
Yes, on holding down the back button it doesn't repeat deletion in iOS.

**Comment by Insality** [comment]
Should be done in 1.1.0
